src/main/java/mate/hwdao/dao/impl/jdbc/DriverDaoJdbcImpl.java

```diff
@@ -0,0 +1,111 @@
+package mate.hwdao.dao.impl.jdbc;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.hwdao.dao.DriverDao;
+import mate.hwdao.dao.exception.DataProcessingException;
+import mate.hwdao.lib.Dao;
+import mate.hwdao.model.Driver;
+import mate.hwdao.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoJdbcImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
+        try (PreparedStatement preparedStatement
+                     = ConnectionUtil.getConnection()
+                .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
let's open the connection as well - in each method
so that it will be closed in the end of the `try` block

Reply:
done